raise ValueError('Json file does not exist!'); in MapEnviroment.py